CMakeLists.txt

```diff
@@ -103,8 +114,35 @@ if(OCIO_BUILD_GPU_TESTS OR OCIO_BUILD_APPS)
         message(WARNING "GLUT not found.")
         set(OCIO_GL_ENABLED OFF)
     endif()
+    set(OCIO_EGL_HEADLESS OFF)
```

Comment:
The code start a new feature detection so you should add an empty line between line 116 and 117.